packages/@n8n/task-runner/src/js-task-runner/freeze-prototypes.ts

```diff
@@ -0,0 +1,20 @@
+import { DateTime, Duration, Interval } from 'luxon';
+import { WorkflowDataProxy, Workflow, Expression } from 'n8n-workflow';
+
+export function freezePrototypes() {
```

Comment:
We should do this only once. And we should also do it before executing any JS code. Since it needs to be done only after certain pyodide initialization is done, this leaves us with 2 options:
1. Always initialize the pyodide stuff when runner starts and then freeze globals, before any code runs. This means we need to load pyodide even if no python code will ever get executed
3. Separate python into its own runner